src/contract-dumps.ts

```diff
@@ -114,12 +114,12 @@ export const makeStateDump = async (cfg: RollupDeployConfig): Promise<any> => {
     deploymentSigner: signer,
     ovmGasMeteringConfig: {
       minTransactionGasLimit: 0,
-      maxTransactionGasLimit: 1_000_000_000,
+      maxTransactionGasLimit: 9_000_000,
```

Comment:
Ah this config option is I believe just for generating the state dump & that makes me lean towards keeping it as an insane value